d/rspec

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+gem=${1%%/*}
+spec=${1#*/}
```

Comment:
This is some bash black magic, what does this do?

I'm not asking for docs, but I'm curious 🤣

Reply:
I forgot I had added this! It allows you to run any spec from the root folder, for example, `d/rspec decidim-core/spec/system/my_system_spec.rb`

Reply:
Oh, nice! Too bad we can't reuse this outside docker, because it's super useful!

Reply:
We can, if we add regular binstubs xD. It'd be a matter of doing the same thing in ruby inside the binstub.

Reply:
I might be missing something, but if we add the equivalent for a non-Docker environment then we need to keep changing the folder to run the tests 🤔 

Unless we modify the main spec_helper so that it knows how to run the tests from the top-level folder... Anyway, that's another discussion 😄

Reply:
Yes, I mean "keep changing the folder when running the tests". Switch folder, run test(s), switch back.

Reply:
Ah, I see 👌